bears/jinja2/Jinja2Bear.py

```diff
@@ -112,11 +112,17 @@ class Jinja2Bear(LocalBear):
 
     VARIABLE_REGEX = re.compile(
         r'(?P<open>{{)(?P<content>.*?)(?P<close>}})')
+    STATEMENT_REGEX = re.compile(
+        r'(?P<open>{%[+-]?)'
+        r'(?P<content>(?![+-]?\s*for)'
+        r'(?![+-]?\s*if)(?![+-]?\s*endfor)'
+        r'(?![+-]?\s*endif).*?)'
```

Comment:
hm can you maybe simplify this to `\s(?!if).*?` or so? You repeat `[+-]` very often and it's already being matched in `<open>`